lib/features/children/family_history/widgets/pending_donation_widget.dart

```diff
@@ -1,85 +1,129 @@
+            decoration: BoxDecoration(
               color: AppTheme.childHistoryPendingLight.withOpacity(0.1),
-              borderRadius: const BorderRadius.all(Radius.circular(20))),
-          child: Row(
-            children: [
-              const SizedBox(width: 15),
-              Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
-                  Text(
-                    '\$${donation.amount.toStringAsFixed(2)} ${locals.childHistoryBy} ${donation.name}',
-                    style: Theme.of(context).textTheme.titleMedium?.copyWith(
-                          color: DonationState.getAmountColor(donation.state),
-                          fontFamily: 'Roboto',
-                          fontWeight: FontWeight.w500,
-                        ),
-                  ),
-                  SizedBox(
-                    width: donation.medium == DonationMediumType.nfc
-                        ? size.width * 0.55
-                        : size.width * 0.75,
-                    child: Text(
-                      donation.organizationName,
-                      maxLines: 2,
-                      overflow: TextOverflow.ellipsis,
-                      softWrap: true,
+              borderRadius: const BorderRadius.all(Radius.circular(20)),
+            ),
+            child: Row(
+              children: [
+                const SizedBox(width: 15),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      '\$${widget.donation.amount.toStringAsFixed(2)} ${locals.childHistoryBy} ${widget.donation.name}',
                       style: Theme.of(context).textTheme.titleMedium?.copyWith(
-                            color: AppTheme.childHistoryPendingDark,
+                            color: DonationState.getAmountColor(
+                                widget.donation.state),
+                            fontFamily: 'Roboto',
+                            fontWeight: FontWeight.w500,
+                          ),
+                    ),
+                    SizedBox(
+                      width: widget.donation.medium == DonationMediumType.nfc
+                          ? size.width * 0.55
+                          : size.width * 0.75,
+                      child: Text(
+                        widget.donation.organizationName,
+                        maxLines: 2,
+                        overflow: TextOverflow.ellipsis,
+                        softWrap: true,
+                        style:
+                            Theme.of(context).textTheme.titleMedium?.copyWith(
+                                  color: AppTheme.childHistoryPendingDark,
+                                  fontFamily: 'Raleway',
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 18,
+                                ),
```

Comment:
Wouldn't be better to have this in a separate widget? if you keep using that fontFamily in more places?